cartesi-rollups/api/typeDefs.graphql

```diff
@@ -71,11 +71,11 @@ type Query {
   "Get input based on its identifier"
   input(index: Int!): Input!
   "Get voucher based on its index"
-  voucher(voucherIndex: Int!, inputIndex: Int!): Voucher!
+  voucher(voucherIndex: Int!, inputIndexWithinEpoch: Int!): Voucher!
   "Get notice based on its index"
-  notice(noticeIndex: Int!, inputIndex: Int!): Notice!
+  notice(noticeIndex: Int!, inputIndexWithinEpoch: Int!): Notice!
   "Get report based on its index"
-  report(reportIndex: Int!, inputIndex: Int!): Report!
+  report(reportIndex: Int!, inputIndexWithinEpoch: Int!): Report!
```

Comment:
These didn't change. Use the following link as a reference https://github.com/cartesi/rollups/blob/main/offchain/graphql-server/schema.graphql